testing/tests/DevExpress.ui.widgets/scrollableParts/scrollable.api.tests.js

```diff
@@ -762,7 +765,7 @@ class ScrollableTestHelper {
 
             if(this._useNative && this._useSimulatedScrollbar) {
                 this.scrollable.update();
-                this._getScrollableContainer().trigger('scroll');
+                this.$container.trigger('scroll');
```

Comment:
this.scrollable.container() ?

Reply:
We need to change a large amount of lines and add this change. Lets do it later.